### PR TITLE
Fixed StacksClient to reference Stacks interface instead of the implementor

### PR DESCRIPTION
**Pull request by Dantheman720**
Fixed the issue with the POM as well as the StacksClient update.

### CODE REVIEW

File: src/main/java/org/jboss/jdf/stacks/parser/Parser.java [hunk 1]
```diff
@@ -51,7 +52,7 @@ public Stacks parse(InputStream is) {
    
         constructor.addTypeDescription(stackDescription);
         Yaml yaml = new Yaml(constructor);
-        Stacks data = (Stacks) yaml.load(is);
+        StacksImpl data = (StacksImpl) yaml.load(is);
```
Comment (gastaldi) [inline src/main/java/org/jboss/jdf/stacks/parser/Parser.java:55]: Is this change necessary ?

Comment (rafabene) [inline src/main/java/org/jboss/jdf/stacks/parser/Parser.java:55]: Good catch. I'm not sure. Didn't test without it.

**Comment by rafabene** [comment]
We will release it under the following GAV: org.jboss.stacks:jboss-stacks-client:1.1.x as part of new organization changes.
